omnibus-arbitrum-mainnet.toml

```diff
@@ -26,11 +26,6 @@ include = [
     "tomls/omnibus-arbitrum-mainnet/collaterals/weth.toml",
     "tomls/omnibus-arbitrum-mainnet/rewards/ARB_rewards_for_WETH_lp.toml",
 
-    # sUSDe
-    "tomls/omnibus-arbitrum-mainnet/oracles/pyth-sUSDe.toml",
-    "tomls/omnibus-arbitrum-mainnet/collaterals/sUSDe.toml",
-    "tomls/omnibus-arbitrum-mainnet/rewards/ARB_rewards_for_sUSDe_lp.toml",
```

Comment:
We haven't deployed mainnet yet so we can still remove these sUSDe deployments. Still bumping version though.